chemberta/finetune/finetune.py

```diff
@@ -131,6 +124,8 @@ def finetune_single_dataset(dataset_name, run_dir):
     dataset_type = get_dataset_info(dataset_name)["dataset_type"]
     if dataset_type == "classification":
         model_class = RobertaForSequenceClassification
+        config.num_labels = len(np.unique(train_labels))
```

Comment:
this is the relevant change

Reply:
theoretically would be better to know ahead of time how many classes we have since this could be slow if we tried it on a big dataset but for now it does the job